docs/metrics.md

```diff
@@ -0,0 +1,466 @@
+
+# TYPE eventstore_proc_contention_count counter
+eventstore_proc_contention_count 297 1688147136862
+
+# TYPE eventstore_gc_pause_duration_max_seconds gauge
+# UNIT eventstore_gc_pause_duration_max_seconds seconds
+eventstore_gc_pause_duration_max_seconds{range="16-20 seconds"} 0.0485873 1688147136862
+```
+
+### Queues
+
+EventStoreDB uses various queues for asynchronous processing for which it also collects different metrics. In addition, EventStoreDB allows users to group queues and monitor them as a unit.
+
+| Time Series | Type       | Description |
+| :---------- | :--------- | :---------- |
+| `eventstore_queue_busy_seconds{queue=<QUEUE_GROUP>}`                                                         | [Counter](#common-types)     | Total time spent processing in seconds, averaged across the queues in the _QUEUE_GROUP_. The rate of this metric is therefore the average busyness of the group during the period (from 0-1 s/s) |
+| `eventstore_queue_queueing_duration_max_seconds{name=<QUEUE_GROUP>,range=<RANGE>}`                           | [RecentMax](#recentmax) | Recent maximum time in seconds for which any item was queued in queues belonging to the _QUEUE_GROUP_. This is essentially the length of the longest queue in the group in seconds |
+| `eventstore_queue_processing_duration_seconds_bucket{message_type=<TYPE>,queue=<QUEUE_GROUP>,le=<DURATION>}` | [Histogram](#common-types)   | Number of messages of type _TYPE_ processed by _QUEUE_GROUP_ group that took less than or equal to _DURATION_ in seconds                         |
+
+`QueueLabels` setting within `telemetryconfig.json` can be used to group queues, based on regex which gets matched on queue names, and label them for metrics reporting. Capture groups are also supported. Message types can be grouped in the same way in the `MessageTypes` setting in `telemetryconfig.json`.
+
+::: note
+Enabling `Queues.Processing` can cause a lot more time series to be generated, according to the `QueueLabels` and `MessageTypes` configuration.
+:::
+
+Example Configuration:
+```
+Queues": {
+	"Busy": true,
+	"Length": true,
+	"Processing": false
+}
+
+"QueueLabels": [
+	{
+		"Regex": "StorageReaderQueue #.*",
+		"Label": "Readers"
+	},
+	{
+		"Regex": ".*",
+		"Label": "Others"
+	}
+]
+```
+
+Example Output:
+```
+# TYPE eventstore_queue_busy_seconds counter
+# UNIT eventstore_queue_busy_seconds seconds
+eventstore_queue_busy_seconds{queue="Readers"} 1.04568158125 1688157491029
+eventstore_queue_busy_seconds{queue="Others"} 0 1688157491029
+
+# TYPE eventstore_queue_queueing_duration_max_seconds gauge
+# UNIT eventstore_queue_queueing_duration_max_seconds seconds
+eventstore_queue_queueing_duration_max_seconds{name="Readers",range="16-20 seconds"} 0.06434454 1688157489545
+eventstore_queue_queueing_duration_max_seconds{name="Others",range="16-20 seconds"} 0 1688157489545
+```
+
+### Status
+
+EventStoreDB can track the current status of the `Node`, the `Index`, and the `Scavenge`
```

Comment:
```suggestion
EventStoreDB tracks the current status of the `Node` role  as well as the `Index`, and the `Scavenge` processes.
```
I'd put that one on top , this is probably yhe first one we want people to actually track

Reply:
Currently, metrics sections are arranged alphabetically. Please let me know if you think some other order is better. (I personally prefer alphabetical order.)

Reply:
Other then that, I have rephrased the statement according to your suggestion.